english/invite-friends/flip/flip.md

```diff
@@ -0,0 +1,3 @@
+You attempt a flip but only manage to screw up and land on your head. Despite being in what is essentially a bounce house made of soft marshmallows, the stunt ends up breaking your neck. You lay paralyzed on the floor, unable to feel the fluffiness of the surface that so recently betrayed you, and your friends leave, bummed that you managed to ruin another party. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.